src/Command/TailwindInitCommand.php

```diff
@@ -24,7 +24,6 @@ class TailwindInitCommand extends Command
 {
     public function __construct(
         private TailwindBuilder $tailwindBuilder,
-        private string $projectDir,
```

Comment:
Technically this may be considered as BC break too :/

Reply:
Hi @bocharsky-bw thanks for your reply. The ability to configure the path of the configuration file makes it possible to deliver the configuration in an encapsulated bundle. I have a Shopware6 project based on Symfony and have created a bundle for a custom theme. Each theme is a separate bundle, hence the requirement.